pkgs/development/python-modules/dash-bootstrap-components/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, dash
+}:
+
+buildPythonPackage rec {
+  pname = "dash-bootstrap-components";
+  version = "1.1.0";
+
+  buildInputs = [
+    dash
+  ];
+
+  # Strange test failure:
+  # TypeError: calling <class 'dash_bootstrap_components._components.CardImg.CardImg'> returned CardImg(None), not a test
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "dash_bootstrap_components"
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-NAdV08pNLlsnRfxnOYeF5nMJL67yCX4olfhYCxrRgW4=";
+  };
```

Comment:
Source should be between version and the used inputs.

Reply:
resolving this since it was posted twice.